src/scripts/main.js

```diff
@@ -1,3 +1,68 @@
 'use strict';
 
 // write code here
+
+const rowAppended = document.querySelector('.append-row');
+const rowRemover = document.querySelector('.remove-row');
+const columnAppended = document.querySelector('.append-column');
+const columnRemover = document.querySelector('.remove-column');
+const field = document.querySelector('.field');
+
+function updateButtonStates() {
+  const trs = field.querySelectorAll('tr');
+  const tds = trs[0] ? trs[0].children : [];
```

Comment:
The code assumes that there is always at least one row in the table. If the table is initially empty, this will cause an error. Consider checking if `trs[0]` exists before accessing its children.